airbyte-cdk/python/README.md

```diff
@@ -140,9 +140,10 @@ and the installation should use your local CDK. Note that the local CDK is injec
 **Note:** if your connector uses a `.dockerignore` file, it cannot have `exclude-all` or `exclude-except` patterns, i.e. the `.dockerignore` must specifically say which files to ignore without using any regex. 
 #### Publishing a new version to PyPi
 
-1. Bump the package version in `setup.py`
-2. Open a PR
-3. An Airbyte member must comment `/publish-cdk dry-run=true` to publish the package to test.pypi.org or `/publish-cdk dry-run=false` to publish it to the real index of pypi.org.
+1. Open a PR
+2. Once it is approved and merge, an Airbyte member must run the `Publish CDK Manually` workflow twice
+   1. Once with `part-to-bump=major|manor|patch` and `dry-run=true` to bump the version and publish to test.pypi.org
```

Comment:
is there a reason not to do the dry run and the actual release as a single step? I don't think I ever want to only run the dry run

Reply:
I think it was just an extra sanity check for what we were trying to pushwas valid since the push to PyPi seems pretty permanent irreversible. I'm not super attached, but I would be fine leaving it just because a few extra minutes on what is already an automated process feels acceptable.

Reply:
Agreed! I'll keep it but we can reassess if this causes any kind of issue later on (like "this is flaky in the new process" or "we are changing things in this pipeline and maintaining this is costly")